src/adapter.cpp

```diff
@@ -937,14 +939,15 @@ void post_tune(adapter *ad) {
 #endif
 #ifndef DISABLE_PMT
     SPid *p_all = find_pid(aid, 8192);
-    if (!p_all || p_all->flags == 3) { // add pids if not explicitly added
+    if (!p_all ||
+        p_all->flags == PID_STATE_DELETED) { // add pids if not explicitly added
         int pid;
         uint32_t ppid = 0;
         int pids[] = {0, 1, 16, 18}; // pids not added by the PMT module
```

Comment:
This array could be declared as a constant somewhere, similar to the PID list used when emulating pids=all. Additionally I think this should check if the adapter is type CI and skip the whole thing.

Reply:
But this can be done in a later PR